documentation/modules/con-kafka-bridge-logging.adoc

```diff
@@ -0,0 +1,74 @@
+z// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka-bridge.adoc
+
+[id='con-kafka-bridge-logging-{context}']
+= Kafka Bridge loggers
+
+Kafka Bridge has a configurable logger for each OpenAPI operation, which are formatted as follows:
+
+[source,yaml,subs="+quotes,attributes"]
+----
+log4j.logger.http.openapi.operation.__<operation-id>__
+----
+
+Where `_<operation-id>_` is the identifier of a specific operation.
+
+Kafka Bridge uses the Apache `log4j` logger implementation.
+
+Here are the operations defined by the OpenAPI specification:
+
+* `createConsumer`
+* `deleteConsumer`
+* `subscribe`
+* `unsubscribe`
+* `poll`
+* `assign`
+* `commit`
+* `send`
+* `sendToPartition`
+* `seekToBeginning`
+* `seekToEnd`
+* `seek`
+* `healthy`
+* `ready`
+* `openapi`
+
+Use the `logging` property to configure loggers and logger levels.
+
+You can set the log levels by specifying the logger and level directly (inline) or use a custom (external) ConfigMap.
+If a ConfigMap is used, it must be placed under under the `log4j.properties` key.
+
+Here we see examples of `inline` and `external` logging.
+
+.Inline logging
+[source,yaml,subs="+quotes,attributes"]
+----
+apiVersion: {KafkaApiVersion}
+kind: KafkaBridge
+spec:
+  # ...
+  logging:
+    type: inline
+    loggers:
+      log4j.logger.io.strimzi.kafka.bridge: "INFO"
+  # ...
+----
+
+.External logging
+[source,yaml,subs="+quotes,attributes"]
+----
+apiVersion: {KafkaApiVersion}
+kind: KafkaBridge
+spec:
+  # ...
+  logging:
+    type: external
+    name: customConfigMap
+  # ...
+----
+
+.Additional resources
+
+* Garbage collector (GC) logging can also be enabled (or disabled). For more information on GC, see xref:ref-jvm-options-deployment-configuration-kafka[]
```

Comment:
Suggested change:

For more information about GC logging, see xref:ref-jvm-options-deployment-configuration-kafka[].

Reply:
Fixed